common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -146,6 +171,13 @@ class LibraryContentFields(object):
         default=1,
         scope=Scope.settings,
     )
+    capa_type = String(
+        display_name=_("Problem Type"),
```

Comment:
@cahrens I'm not sure what the question here, so I'll give you a broad description of what's happening hoping to answer it.

`capa_type` is a singular value (i.e. not comma separated list) that contains a tag name of response type. If it's *not* "Any Type" it's used to filter library contents by problem type using the following rules:

1. All library contents that are not CAPA problems are immediately filtered out
2. **All** response tags are extracted from CAPA problems. If **any** match selected `capa_type` problem is included in the result set.

So, say we have four blocks in the library:

1. HTML Block
2. CAPA problem with multiple choice input
3. CAPA problem with select input
4. CAPA problem with both multiple choice and select inputs

Filtering by `capa_type` we'll get the followin gresults:

1. `capa_type` == "Any Type": 1,2,3,4
2. `capa_type` == 'Multiple Choice": 2,4
3. `capa_type` == 'Select Option": 3,4
4. `capa_type` == 'Matlab': *none*